src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    margin: 16px 3px 0;
+    width: 160;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-text-color;
+    margin-top: 40px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $sec-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 4px;
+  }
+
+  &__raiting {
+    color: $main-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 16px;
+    height: 16px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: $sec-text-color;
+    font-size: 12px;
+    font-weight: 400;
+    margin-top: 24px;
+
+    &_value {
+      color: $main-text-color;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+  &__btn {
+    background: $btn-color;
+    color: white;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    font-size: 14px;
+    font-weight: 700;
+    text-transform: uppercase;
+    padding: 0;
+    margin-top: 16px;
+    font-family: Roboto, sans-serif;
```

Comment:
It's your main font that you have specified to body, so you don't need to set it here